src/main/java/calculator/Operand.java

```diff
@@ -0,0 +1,17 @@
+package calculator;
+
+public class Operand extends FormulaElement {
+    private final double operand;
+
+    public Operand(String input) {
+        operand = Double.parseDouble(input);
```

Comment:
```suggestion
        this(Double.parseDouble(input));
```

생성자가 여러 개라면 생성 시 로직이 변경됐을 때 모든 생성자를 변경해야하는 일이 발생할 수 있어요.
최종적인 초기화는 하나의 생성자에서 처리해주세요!

Reply:
이 부분에 하나 더 여쭤보고 싶은게 있습니다.

`Double.parseDouble()`은 NumberFormatException 을 throw하는데, 만약 이 예외처리로는 설명이 불충분하다고 느끼면 이 부분을 try~ catch~ 구문을 사용해서 커스텀 예외를 발생시키는 것도 하나의 방법이 되는지 궁금합니다!

Reply:
네! 말씀하신 상황에서 커스텀 예외를 발생하는 것은 좋은 방법입니다. 👍